DFC.ServiceTaxonomy.GraphSync/GraphSyncers/Parts/Flow/FlowPartGraphSyncer.cs

```diff
@@ -27,8 +28,16 @@ public class FlowPartGraphSyncer : IContentPartGraphSyncer
             _contentFieldsGraphSyncer = contentFieldsGraphSyncer;
         }
 
+        public async Task AllowSync(JArray? contentItems, IGraphMergeContext context, IAllowSyncResult allowSyncResult)
+        {
+            await Task.WhenAll(
+                _flowPartEmbeddedContentItemsGraphSyncer.AllowSync(contentItems, context, allowSyncResult),
+                _contentFieldsGraphSyncer.AllowSync(contentItems, context, allowSyncResult));
```

Comment:
Is it worth only executing this WhenAll if contentItems is not null? Only a very minor point

Reply:
Oops, this wasn't implementing the correct `AllowSync` signature. Fixed.